### PR TITLE
 Dataframe: Replace old filehandling methods with pandas dataframe

### PR DESCRIPTION
**Pull request by ananthanandanan**
- Changed the tid_store existing  txt format to CSV format.
- Add datetime field to parse latest tid.
- This commit fixes issue #36

### CODE REVIEW

**Comment by ananthanandanan** [comment]
For this method it is better to manually use tid_store object from separate file. Since to_csv() takes path.

File: twitter_bot.py [hunk 1]
```diff
@@ -59,13 +70,11 @@ def isSafeKeyword(self, tweetText):
 
     def nawab_get_id(self):
```
Comment (Aniketh01) [inline twitter_bot.py:68]: Can you rename the function and add comment in the top of the function to denote what does this exactly does?

**Comment by iammarco11** [comment]
Now this looks fine, the buttons all work fine :)
@Aniketh01 I guess you can do the final review.

**Review by Aniketh01 — COMMENTED** [review]
For Instance in line 131 of Twitter bot, it's index, Id in tid.iterrows():

And it's used as Id.id, which looks bad. Perhaps have it as tid_store.id?

**Comment by Aniketh01** [comment]
Also, have date_time or something for datetime header.

**Review by Aniketh01 — APPROVED** [review]

**Pull request merged**